tests/mobly/test_runner_test.py

```diff
@@ -179,7 +180,7 @@ def test_run_two_test_classes(self, mock_get_all, mock_list_adb,
         tr = test_runner.TestRunner(mock_test_config,
                                     [('IntegrationTest', None),
                                      ('IntegrationTest', None)])
-        tr.run()
+        tr.run([IntegrationTest.IntegrationTest])
```

Comment:
Seems like all the unit tests specify a single class?
Should we add a test for when multiple classes are specified?